### PR TITLE
EPMRPP-80698 || Open source plan

### PR DESCRIPTION
**Pull request by AmsterGet**
Open Source plan added.
`ComparisonTable` and `ComparisonList` components created to improve readability of `PlansBlock` component.

### CODE REVIEW

File: src/react-components/pages/pricing-page/plans-block/comparison/comparison-list/comparisonList.jsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  });
+
+  return <div className={cx('comparison-list-wrapper', { opened: isOpen })}>
+    <div className={cx('comparison-list')}>
+      {plans}
+      <div className={cx('note')}>Payment is made quarterly</div>
+      <Terms className={cx('terms')} />
+    </div>
+  </div>;
+}
```
Comment (tr1ble) [inline src/react-components/pages/pricing-page/plans-block/comparison/comparison-list/comparisonList.jsx]: Please add `;`

File: src/react-components/pages/pricing-page/plans-block/comparison/comparison-list/comparisonList.jsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+ComparisonList.propTypes = {
+  plansData: PropTypes.array.isRequired,
+  planType: PropTypes.shape({
+    planCompareTableTitles: PropTypes.array.isRequired,
+  }).isRequired,
+  isOpen: PropTypes.bool.isRequired,
+}
```
Comment (tr1ble) [inline src/react-components/pages/pricing-page/plans-block/comparison/comparison-list/comparisonList.jsx]: Please add `;`

**Review by tr1ble — CHANGES_REQUESTED** [review]

File: src/react-components/pages/pricing-page/plans-block/comparison/comparison-table/comparisonTable.jsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    </td>
+  );
+
+  return (
+    <Table
+      className={cx('comparison-table', { opened: isOpen })}
+      data={{ headers, rows, footer }}
+    />
+  );
+}
```
Comment (tr1ble) [inline src/react-components/pages/pricing-page/plans-block/comparison/comparison-table/comparisonTable.jsx]: Please add `;`

**Review by tr1ble — CHANGES_REQUESTED** [review]

File: src/react-components/pages/pricing-page/plans-block/comparison/comparison-list/comparisonList.jsx [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+            preposition = ' ';
+          }
+          break;
+        default:
+          break;
+      }
+
+      return <div key={index} className={cx('plan-row', { disable: !option })}>
+        <div className={cx('row-status')}>
+          <i/>
```
Comment (tr1ble) [inline src/react-components/pages/pricing-page/plans-block/comparison/comparison-list/comparisonList.jsx]: Please add space before `/>`

**Review by tr1ble — CHANGES_REQUESTED** [review]

File: src/react-components/pages/pricing-page/plans-block/data.js [hunk 5]
```diff
@@ -167,30 +205,29 @@ const plansData = [
       retention: 'Unlimited',
       support: {
         isMultiLine: true,
-        value: 'Dedicated\n' +
-        'specialist',
+        value: 'Dedicated\n' + 'specialist',
```
Comment (tr1ble) [inline src/react-components/pages/pricing-page/plans-block/data.js]: Please fix lint here to fix pipeline

**Review by tr1ble — CHANGES_REQUESTED** [review]

**Review by tr1ble — APPROVED** [review]

**Review by Bam6ycha — APPROVED** [review]

**Pull request merged**